server/configNotifiers/notifiers/postgres.js

```diff
@@ -0,0 +1,47 @@
+import PGPubSub from "pg-pubsub";
+import { info } from "../../logger";
+
+/**
+ * A pubsub implementation that uses Postgres notify/listen feature and the
+ * pg-pubsub lib (https://github.com/voxpelli/node-pg-pubsub)
+ */
+export class PostgresNotifier {
```

Comment:
❤️ Before your most recent commit, I was going to suggest using a class here. This is nice